Azure/communication-ui-library-android
# [Feature] Display version and build number label in the Demo App

## Purpose

* Display version and build number label in the Demo App

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Test the code

```
Build demo app using app center and observe label at the bottom.
If build in debug (locally) version is replaced with 'Debug'
```

## What to Check
Verify that the following are valid
* Have Demo App started

## Other Information
